packages/prisma/schema.prisma

```diff
@@ -1586,3 +1587,17 @@ model AttributeToUser {
 
   @@unique([memberId, attributeOptionId])
 }
+
+model Translation {
+  id             String     @id @default(cuid())
+  sourceText     String     @db.Text
```

Comment:
Do we really need the sourceText since ideally this always points to the current event type description? 

This is potentially taking a lot of unneeded DB space

Reply:
addressed